events/syscall/create_hidden_file_or_directory.go

```diff
@@ -21,14 +21,21 @@ import (
 )
 
 var _ = events.Register(
-	CreateHiddenFileOrDirectory,
+	CreateHiddenFilesOrDirectories,
```

Comment:
Can you also rename the file name to `create_hidden_file_or_directories.go`, please?

Reply:
i've renamed it to create_hidden_file**s**_or_directories.go